e3sm_to_cmip/lib.py

```diff
@@ -735,8 +735,14 @@ def load_axis(data, levels=None, has_time=True):
 
         axis_bnds = levels.get("e3sm_axis_bnds")
         if axis_bnds:
+            cell_bounds = data[axis_bnds]
+            # i.g. handler clcalipso returns xarray dataarray
+            if not isinstance(data[axis_bnds], np.ndarray):
+                cell_bounds = data[axis_bnds].values
+                coord_vals = coord_vals.values
+                 
             lev = cmor.axis(
-                name, units=units, cell_bounds=data[axis_bnds].values, coord_vals=coord_vals.values
+                name, units=units, cell_bounds=cell_bounds, coord_vals=coord_vals
```

Comment:
> It is worth to check why the axes objects have various type, but will not address in this PR

I believe `data` is an `xr.Dataset`, which stores coordinates and bounds as `xr.DataArray`. 

I noticed code directly below this section was updated to extract the numpy array from the `xr.DataArray` objects using `.values`:
https://github.com/E3SM-Project/e3sm_to_cmip/blob/4f7a0a45bd8617d9350d48ca192cdfe400c76b90/e3sm_to_cmip/lib.py#L744-L757

This leads me to believe that this section of code related to `lev` was _not_ updated (probably an oversight) until now.